<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1098</width>
    <height>660</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="Line" name="line">
   <property name="geometry">
    <rect>
     <x>23</x>
     <y>66</y>
     <width>1091</width>
     <height>16</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>120</y>
     <width>141</width>
     <height>431</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QPushButton" name="add_person_btn">
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="text">
       <string>add_person_btn</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton_6">
      <property name="text">
       <string>PushButton</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton_5">
      <property name="text">
       <string>PushButton</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton_4">
      <property name="text">
       <string>PushButton</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton_3">
      <property name="text">
       <string>PushButton</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton">
      <property name="text">
       <string>PushButton</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="home_icon">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>20</y>
     <width>54</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>1000</x>
     <y>20</y>
     <width>54</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QFrame" name="home_frame">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>120</y>
     <width>1</width>
     <height>1</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QPushButton" name="pushButton_7">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>150</y>
      <width>87</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>PushButton</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_8">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>150</y>
      <width>87</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>PushButton</string>
    </property>
   </widget>
  </widget>
  <widget class="QFrame" name="add_person_frame">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>120</y>
     <width>1</width>
     <height>1</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="frameShape">
    <enum>QFrame::Box</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Sunken</enum>
   </property>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>73</y>
      <width>54</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Nom</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="F_name">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>65</y>
      <width>220</width>
      <height>33</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>120</y>
      <width>54</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Prenom</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="L_name">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>112</y>
      <width>220</width>
      <height>33</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="cne">
    <property name="geometry">
     <rect>
      <x>141</x>
      <y>160</y>
      <width>220</width>
      <height>33</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>168</y>
      <width>54</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>C.I.N</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>210</y>
      <width>54</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Sexe</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QRadioButton" name="femme">
    <property name="geometry">
     <rect>
      <x>154</x>
      <y>210</y>
      <width>100</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Femme</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="homme">
    <property name="geometry">
     <rect>
      <x>274</x>
      <y>210</y>
      <width>100</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Homme</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>250</y>
      <width>70</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Naissance</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QDateEdit" name="birth_date">
    <property name="geometry">
     <rect>
      <x>145</x>
      <y>240</y>
      <width>170</width>
      <height>27</height>
     </rect>
    </property>
    <property name="wrapping">
     <bool>true</bool>
    </property>
    <property name="buttonSymbols">
     <enum>QAbstractSpinBox::UpDownArrows</enum>
    </property>
    <property name="accelerated">
     <bool>false</bool>
    </property>
    <property name="displayFormat">
     <string>dd-MM-yyyy</string>
    </property>
    <property name="calendarPopup">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="age_counter">
    <property name="geometry">
     <rect>
      <x>325</x>
      <y>242</y>
      <width>31</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>00</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>290</y>
      <width>110</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Situation Familial</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QComboBox" name="familly_state">
    <property name="geometry">
     <rect>
      <x>142</x>
      <y>285</y>
      <width>220</width>
      <height>25</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>Celibataire</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Marie</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Divorce</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Separe</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>veuf</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>16</x>
      <y>341</y>
      <width>110</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Nomber d'enfant</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="children_num">
    <property name="geometry">
     <rect>
      <x>146</x>
      <y>330</y>
      <width>210</width>
      <height>33</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_10">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>70</y>
      <width>110</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Adresse</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="address">
    <property name="geometry">
     <rect>
      <x>611</x>
      <y>56</y>
      <width>210</width>
      <height>33</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_11">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>110</y>
      <width>110</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>TEL</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="tel">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>101</y>
      <width>210</width>
      <height>33</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_12">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>150</y>
      <width>120</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Type d'assurance</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QComboBox" name="assurance_type">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>150</y>
      <width>213</width>
      <height>27</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>Rien</string>
     </property>
    </item>
   </widget>
   <widget class="QTextEdit" name="note">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>210</y>
      <width>240</width>
      <height>111</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_13">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>250</y>
      <width>120</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Remarque</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="add_person_save_btn">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>370</y>
      <width>87</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>Enregistrer</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_9">
    <property name="geometry">
     <rect>
      <x>740</x>
      <y>370</y>
      <width>87</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>annuler</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>pushButton_9</sender>
   <signal>clicked()</signal>
   <receiver>F_name</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>955</x>
     <y>501</y>
    </hint>
    <hint type="destinationlabel">
     <x>429</x>
     <y>210</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_9</sender>
   <signal>clicked()</signal>
   <receiver>L_name</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>971</x>
     <y>505</y>
    </hint>
    <hint type="destinationlabel">
     <x>516</x>
     <y>251</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_9</sender>
   <signal>clicked()</signal>
   <receiver>cne</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>976</x>
     <y>495</y>
    </hint>
    <hint type="destinationlabel">
     <x>534</x>
     <y>305</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_9</sender>
   <signal>clicked()</signal>
   <receiver>children_num</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>975</x>
     <y>508</y>
    </hint>
    <hint type="destinationlabel">
     <x>517</x>
     <y>472</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_9</sender>
   <signal>clicked()</signal>
   <receiver>address</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>946</x>
     <y>495</y>
    </hint>
    <hint type="destinationlabel">
     <x>898</x>
     <y>206</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_9</sender>
   <signal>clicked()</signal>
   <receiver>tel</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>996</x>
     <y>491</y>
    </hint>
    <hint type="destinationlabel">
     <x>964</x>
     <y>242</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_9</sender>
   <signal>clicked()</signal>
   <receiver>note</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>999</x>
     <y>494</y>
    </hint>
    <hint type="destinationlabel">
     <x>958</x>
     <y>398</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
